app/assets/stylesheets/adopt.css.scss

```diff
@@ -35,3 +36,58 @@
     font-weight: normal;
   }
 }
+
+ul.list-of-rats {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+}
+
+ul.list-of-rats li{
+  display: inline-block;
+  height: 150px;
+  margin: 100px 1em 1em 0;
+  position: relative;
+  width: 150px;
+}
+
+span.text-content {
+  background: rgba(0,0,0,0.5);
+  color: $blue;
+  cursor: pointer;
+  display: table;
+  height: 100px;
+  left: 0;
+  position: absolute;
+  top: 0;
+  width: 150px;
+  opacity: 0;
+  -webkit-transition: opacity 500ms;
+  -moz-transition: opacity 500ms;
+  -o-transition: opacity 500ms;
+  transition: opacity 500ms;
+}
+
+span.text-content span {
+  display: table-cell;
+  text-align: center;
+  vertical-align: middle;
+}
+
+ul.list-of-rats li:hover span.text-content {
```

Comment:
Merge rule `ul.list-of-rats li:hover span.text-content` with rule on line 40<br>Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3